src/components/experiment-tracking/time-series/time-series.scss

```diff
@@ -116,3 +116,7 @@
   text-anchor: end;
   transform: translate(-10px, -4px);
 }
+
+.brush .selection {
+  y: -1;
+}
```

Comment:
What's this doing exactly?

Reply:
This is why I changed the default style for the brush, the bottomLine is under the xAxis:

<img width="613" alt="Screenshot 2022-12-07 at 13 31 17" src="https://user-images.githubusercontent.com/88193161/206180043-4b78867a-9366-4421-adc0-58f03de71b68.png">